app/scripts/modules/core/instance/instance.write.service.js

```diff
@@ -12,12 +12,10 @@ module.exports = angular
     function terminateInstance(instance, application, params={}) {
       params.type = 'terminateInstances';
       params.instanceIds = [instance.instanceId];
-      params.serverGroup = instance.serverGroup;
-      params.launchTimes = [instance.launchTime];
+      params.serverGroupName = instance.serverGroup;
```

Comment:
I never saw anywhere in the code where just `serverGroup` was used, so I changed it to the expected `serverGroupName`, which Orca looks for.